frontend/src/lib/components/widgets/CameraInput/CameraInput.tsx

```diff
@@ -569,15 +586,12 @@ class CameraInput extends React.PureComponent<Props, State> {
     this.props.uploadClient
       .uploadFile(
         this.props.element,
-        // TODO(vdonato): Use the file upload URL received from the server.
-        "/_stcore/upload_file",
+        fileURLs.uploadUrl as string,
         file,
         e => this.onUploadProgress(e, uploadingFileInfo.id),
         cancelToken.token
       )
-      .then(newFileId =>
-        this.onUploadComplete(uploadingFileInfo.id, newFileId)
-      )
+      .then(newFileId => this.onUploadComplete(uploadingFileInfo.id, fileURLs))
```

Comment:
nit: I think `newFileId` here is always going to be `undefined` or `null`, so the arg for this arrow function can probably be dropped